packages/forms-web-app/src/middleware/check-path-allowed.js

```diff
@@ -14,7 +14,9 @@ const checkPathAllowed = (req, res, next) => {
 		'/appeal-householder-decision/email-address-confirmed',
 		'/appeal-householder-decision/confirm-email-address',
 		'/appeal-householder-decision/list-of-documents',
-		'/appeal-householder-decision/application-saved'
+		'/appeal-householder-decision/application-saved',
+		'/full-appeal/submit-appeal/enter-code/^[0-9a-fA-F]{8}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{12}$',
+		'/enter-code/^[0-9a-fA-F]{8}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{4}\\b-[0-9a-fA-F]{12}$'
```

Comment:
`skipMiddlewareIfFinalComments` is already called on `checkPathAllowed` for `/full-appeal` routes - you could add 'submit-appeal/enter-code` to the existing `skipMiddleware` function and rename it to reflect the fact it's no longer just applying to final comments.

iirc, the checkPathAllowed middleware is there to check pages where a user doesn't need to have an appeal in the session - if they to access such a page and don't have an appeal, it will direct them to before you start.

Reply:
These path entries have been removed now